spec/support/authentication_helpers.rb

```diff
@@ -4,30 +4,20 @@ module AuthenticationHelpers
   mattr_accessor :username, :password, :token
 
   def authenticate!(start_url:, expected_text:)
+    ensure_username! # sunet is needed by some tests, even if the user doesn't have to enter user/pass for Stanford web authN
```

Comment:
after some recent changes, i was never prompted for my sunet, since i don't go through the username/pass login form anymore.  and since i don't store my login info locally, the ETD spec would get a blank username when trying to build the fake registrar XML).